modules/internal/ChapelAutoAggregation.chpl

```diff
@@ -102,18 +103,27 @@ module ChapelAutoAggregation {
       const bufferSize = dstBuffSize;
       const myLocaleSpace = LocaleSpace;
       var opsUntilYield = yieldFrequency;
-      var lBuffers: [myLocaleSpace] [0..#bufferSize] aggType;
+      var lBuffers: c_ptr(c_ptr(aggType));
       var rBuffers: [myLocaleSpace] remoteBuffer(aggType);
-      var bufferIdxs: [myLocaleSpace] int;
+      var bufferIdxs: c_ptr(int);
 
       proc postinit() {
+        lBuffers = c_malloc(c_ptr(aggType), numLocales);
```

Comment:
`numLocales` -> `myLocaleSpace.size` ?

Reply:
There are other `numLocales`s that need changing for better maintainability, I think.

Reply:
Note that `myLocaleSpace` was just a copy of `LocaleSpace` that was added back when we used to adjust the reference count for `LocaleSpace` for every array declared over it. Since your const domain work I expect we could go back to `LocaleSpace`.

For your suggestion though, I'm just copying over the code from Arkouda, so I don't want them to get out of sync. If we want to make these changes I'd want to do it in Arkouda first and I'm inclined to punt on that until after the release due to limited time on my end.

Reply:
I see. I'm OK with that.

I was thinking that in the future we might want aggregators to have the equivalent of `targetLocales` that we somehow get from the forall's domain's targetLocales. I don't think that's gonna happen soon, and this is probably the least troublesome thing if we want to do that.